testplan/testing/multitest/result.py

```diff
@@ -1442,6 +1442,42 @@ def markdown(self, message, description=None, escape=True):
         _bind_entry(entry, self)
         return entry
 
+    def html(self, code, description="Embed HTML"):
+        """
+        Create a markdown message entry without escape, can be used for
+        providing additional context related to test steps.
+
+        :param code: HTML code string. Tag <script> will not be executed.
+        :type code: ``str``
+        :param description: Text description for the assertion.
+        :type description: ``str``
+        :return: ``True``
+        :rtype: ``bool``
+        """
+        return self.markdown(code, description=description, escape=False)
+
+    def codelog(self, code, language="python", description="Code log"):
```

Comment:
This can be better if would be implemented just on top of SyntaxHighlighter instead of markdown as an intermediary. For this level of code log, I would go for a tip/suggestion/example next to the markdown in the documentation. 

With a full blown implementation on top of SyntaxHighlighter, many fine tune possibility can be put into the hand of the user.